lib/python/picongpu/plugins/plot_mpl/energy_histogram_visualizer.py

```diff
@@ -53,6 +60,12 @@ def _update_plt_obj(self, idx):
         Implementation of base class function.
         """
         counts, bins, iteration, dt = self.data[idx]
+
+        if np.all(counts == 0.):
+            warn(f"All counts were 0 for {self.sim_labels[idx]}. "
+                 "Log-plot will not be updated!")
```

Comment:
same python2 issue here 
```suggestion
            warn("All counts were 0 for {}. ".format(self.sim_labels[idx]) +
                 "Log-plot will not be updated!")
```